flow/record/fieldtypes/__init__.py

```diff
@@ -283,6 +283,9 @@ def __repr__(self):
         result = str(self)
         return result
 
+    def __hash__(self):
+        return super.__hash__(self)
```

Comment:
```suggestion
        return super().__hash__(self)
```

But I think it is better to not depend on the MRO and use the hash of the thing that actually contains the data:

```suggestion
        return _dt.__hash__(self)
```

Reply:
Changed it to the latter ;)